From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: remove inlude asm/page.h

This patch removes the include of asm/page.h which is not present
on toolchains based on to userspace exported kernel headers.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 mount/swapargs.h |    1 -
 1 file changed, 1 deletion(-)

Index: util-linux-2.13-pre7/mount/swapargs.h
===================================================================
--- util-linux-2.13-pre7.orig/mount/swapargs.h
+++ util-linux-2.13-pre7/mount/swapargs.h
@@ -1,3 +1,2 @@
 #define SWAPON_HAS_TWO_ARGS
-#include <asm/page.h>
 #include <sys/swap.h>
